### DISCUSSION TITLE
XSD - sémák

### DISCUSSION THREAD

**Discussion by vectorkovacspeter** [nav-gov-hu/eVAT#8] [Kérdések és válaszok - Gépi interfész] [answered]
Tisztelt illetékesek,

mikorra várhatóak sémadefiníciók akár XSD akár más formátumban?

Az online számla rendszer kapcsán szerintem látható volt, hogy a fejlesztői diskurzus, csak ez után tud valójában elindulni majd.

**Comment by NTCA-tax** [comment, marked as answer]
Az eÁfa M2M rendszernél nem úgy indulunk, mint az Online Számla esetén. Tehát nem egy közel kész specifikációt adunk közre, és mindenki rohan a megvalósítással, hanem először egy tervezetet publikálunk, amelyre megvárjuk a javaslatokat, véleményeket. Mindezek figyelembe vételével fogjuk a specifikációt kialakítani. Ugyanakkor specifikáció szintjén már jelentős működésbeli változás nem nagyon várható. A fejlesztői és ügyféli igények, javaslatok alapján fogjuk véglegesíteni a rendszer scope-ját, a jelenlegi véleményezés ezért fontos. A specifikáció és XSD-k publikálása egy következő lépést jelent a mi oldalunkon is.
